uber/templates/regform.html

```diff
@@ -515,6 +513,21 @@
     </div>
 </div>
 
+{% if c.PAGE_PATH not in ['/preregistration/form', '/preregistration/post_form'] %}
+  <div class="form-group">
+    <label for="requested_hotel_info" class="col-sm-3 control-label optional-field">Request Hotel Info</label>
+    <div class="checkbox col-sm-9">
+      <label for="requested_hotel_info" class="checkbox-label">
+        <input type="checkbox" name="requested_hotel_info" id="requested_hotel_info" value="1"
+          {% if attendee.requested_hotel_info or (group and group.requested_hotel_info) %}checked{% endif %}>
+        Yes, tell me when hotels begin accepting reservations
+      </label>
+    </div>
+  </div>
+{% else %}
+  <input type="hidden" name="requested_hotel_info" value="{{ '1' if (attendee.is_new and not edit_id) or attendee.requested_hotel_info else '0' }}" />
```

Comment:
`and` binds tighter than `or`, so you technically shouldn't need these parentheses.